pkg/operator2/operator.go

```diff
@@ -284,3 +293,13 @@ func getPrefixFilter() controller.Filter {
 		DeleteFunc: prefix,
 	}
 }
+
+func (c *authOperator) getOperatorDeployment() (*appsv1.Deployment, error) {
+	deployments := c.deployments.Deployments(targetNameOperator)
+	operator, err := deployments.Get(targetNameOperator, metav1.GetOptions{})
```

Comment:
return here

Reply:
Made it a one-line function, maybe still worth as a shortcut, although we're only using it in this one spot.

Reply:
Inlined